dts/arm/st/l5/stm32l5.dtsi

```diff
@@ -246,6 +246,168 @@
 			status = "disabled";
 			label = "RNG";
 		};
+
+		timers1: timers@50012c00 {
```

Comment:
0x50012c00 is the address for secure world. but Zephyr is running in non secure.
```suggestion
		timers1: timers@40012c00 {
```
The same for all timers

Reply:
Resolved.